cookbook/how_to_apply_patch.rst

```diff
@@ -0,0 +1,122 @@
+.. index::
+    single: Patch
+
+How to apply patch
+==================
+
+The patch file name supports the following naming convention: {package}-{version_to_apply}.patch.
+For example, platform-1.9.2.patch.
+
+**1** Create backups of your Database and Code.
+
+**2** Stop the cron tasks
```

Comment:
Should be checked is it necessary to stop the cron in case when we put system to the maintenance mode with app/console lexik:maintenance:lock. If I remember correctly cron will not perform any actions while maintenance mode is turned on.